VMEncryption/main/Common.py

```diff
@@ -61,8 +61,12 @@ class CommonVariables:
     CVM LUKS2 header token
     """
     cvm_ade_vm_encryption_token_id = 5
+    cvm_ade_vm_encryption_backup_token_id = 6
```

Comment:
Pls add comment for its purpose. # Used to store the previous token value during KEK rotation.

Reply:
done